catlin/pkg/validator/content_validator.go

```diff
@@ -51,10 +60,27 @@ func (v *ContentValidator) Validate() Result {
 	}
 
 	annotations := res.GetAnnotations()
+
+	resourceCategories := strings.Split(annotations[consts.CategoryAnnotation], ",")
```

Comment:
Should not this inside else block later?

Reply:
or here return error if resource categories is empty

Reply:
updated !